packages/dd-trace/src/profiling/util.js

```diff
@@ -8,6 +8,55 @@ function maybeRequire (id) {
   }
 }
 
+function eachSeries (collection, iteratee, callback) {
+  eachOfSeries(collection, (item, index, callback) => iteratee(item, callback), callback)
+}
+
+function eachOfSeries (collection, iteratee, callback = () => {}) {
+  const results = []
```

Comment:
If this is in a hot path, you may want to initialize the array to the appropriate length immediately, and then set at indicies rather than pushing (i.e. resizing the array all the time).